python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -62,6 +62,9 @@ def __init__(self, namePattern='', localDir='', **kwds):
         else:
             logger.error("Unkown type: %s . Cannot Create LocalFile from this!" % type(namePattern))
 
+        if localDir:
+            self.localDir = localDir
```

Comment:
Isn't `self.localDir` already set by the `if`/`elif`/`else` branches above? The only one that doesn't cover it is the first so perhaps it should be moved there.